sparse/usr/bin/droid/droid-hal-shutdown.sh

```diff
@@ -62,7 +62,7 @@ WAIT=1
 get_pids
 MAX_WAIT=5
 # -gt 1 because droid-hal-init is also in this cgroup
-while [ $NUM_PIDS -gt 1 -a $WAIT -lt $MAX_WAIT ]; do
```

Comment:
-a isn't ambiguous but not POSIX.

Reply:
`-a` can be ambiguous, although not in this case. Better to not use it since the updated syntax is easier to read.